src/test/java/plugins/MatrixAuthPluginTest.java

```diff
@@ -48,17 +57,27 @@ public void helloWorld() throws Exception {
         jenkins.login().doLogin("alice");
 
         FreeStyleJob j = jenkins.jobs.create();
+
         j.save();
 
+        jenkins.logout();
+
         // if we login as Bob, he shouldn't see the job
         jenkins.login().doLogin("bob");
-        assertNull(getElement(by.href("job/"+j.name+"/")));
 
-        // contorl assertion: alice shoudl see the link
+        // check for job's existence
+        driver.get(controller.getUrl() + "job/" + j.name + "/");
```

Comment:
@olivergondza I can't use `Matchers.pageObjectExists` because to access the url we need to login first, so `Matchers.pageObjectExists`receives a 403 even in the case the job exist. I've switched to `j.open()` because i believe modifying `Matchers.pageObjectExists` to take into account login credentials is out of context for this PR